stage0/src/zero_page.rs

```diff
@@ -15,68 +15,88 @@
 //
 
 use crate::{cmos::Cmos, fw_cfg::FwCfg};
-use core::{
-    ffi::CStr,
-    mem::{size_of, MaybeUninit},
-};
+use core::{ffi::CStr, mem::size_of};
 use oak_linux_boot_params::{BootE820Entry, BootParams, E820EntryType};
 use oak_sev_guest::io::PortFactoryWrapper;
 
-#[link_section = ".boot.zero_page"]
-static mut BOOT_ZERO_PAGE: MaybeUninit<BootParams> = MaybeUninit::uninit();
+#[repr(transparent)]
+#[derive(Debug)]
+pub struct ZeroPage {
+    inner: BootParams,
```

Comment:
Why do you call this `inner` and not `bootParams`?

Reply:
No particular reason beyond that seems to be a common pattern I've seen elsewhere.